src/lib/ndpi_main.c

```diff
@@ -1154,7 +1154,7 @@ static void ndpi_init_protocol_defaults(struct ndpi_detection_module_struct *ndp
   ndpi_set_proto_defaults(ndpi_str, 0 /* encrypted */, NDPI_PROTOCOL_FUN, NDPI_PROTOCOL_DOH_DOT,
 			  "DoH_DoT", NDPI_PROTOCOL_CATEGORY_NETWORK /* dummy */,
 			  ndpi_build_default_ports(ports_a, 853, 0, 0, 0, 0) /* TCP */,
-			  ndpi_build_default_ports(ports_b, 784, 0, 0, 0, 0) /* UDP */);
+			  ndpi_build_default_ports(ports_b, 853, 0, 0, 0, 0) /* UDP */);
```

Comment:
Isn't better to keep the old port together with the new one? I guess older protocol versions can be still seen in the wild.